src/components/tabs/Tab.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  export let key;
+  import { getContext } from "svelte";
+
+  const store = getContext("activeTab");
+
+  function selectTab() {
+    $store = key;
+  }
+</script>
+
+<style>
+  .title {
+    /* border: solid lightgray 2px; */
+    border: 1px solid #dee2e6;
+    border-bottom: none;
+    border-radius: 0.5rem 0.5rem 0 0;
+    display: inline-block;
+    padding: 1em;
+    color: black;
+    background-color: white;
+  }
+  .title.active {
+    color: black;
+    background-color: #a9b4c2;
+  }
+  .title:not(.active):hover {
+    cursor: pointer;
+    color: black;
+    background-color: #d3d5da;
+  }
```

Comment:
Can you use tailwind for some of these properties? This will make these styles a little easier to read/understand/change.

These directives would be relevant here:

https://tailwindcss.com/docs/border-width
https://tailwindcss.com/docs/border-radius
https://tailwindcss.com/docs/customizing-colors
https://tailwindcss.com/docs/padding